app/resources/forms.py

```diff
@@ -15,18 +18,22 @@ class ResourceForm(Form):
         InputRequired(),
     ])
     website = StringField('Website')
-    street_address = StringField('Street Address', validators=[
+
+    street_number = IntegerField('Street Number', validators=[
         InputRequired()
     ])
-    city = StringField('City', validators=[
+    route = StringField('Street Address', validators=[
```

Comment:
@anoshaminai Yes. The way the Autocomplete example worked was that the divs would be named for the fields that the API recognizes (see JS at bottom of add.html).